client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
```

Comment:
I believe it would show 0.3 after they move the heavier item away: `+( 3.3 - 3 ).toFixed( n )` for 0 < n < 16. I can see why a separate PR might be appropriate, since it exists independently of this thread's design discussion.